### ISSUE TITLE
translate documentation into Japanese / 文書を日本語に翻訳する

### ISSUE THREAD

**Issue by 0xadada** [mirai-audio/system-font-i18n-css#6]
# translate documentation into Japanese / 文書を日本語に翻訳する

## En

* [ ] Translate the "About" section of [index.html](https://github.com/mirai-audio/system-font-i18n-css/blob/master/index.html) into Japanese.
* [ ] Translate the "Usage" section of [index.html](https://github.com/mirai-audio/system-font-i18n-css/blob/master/index.html) into Japanese.
* [ ] Translate the "Support" section of [index.html](https://github.com/mirai-audio/system-font-i18n-css/blob/master/index.html) into Japanese.
* [ ] Translate the [README](https://github.com/mirai-audio/system-font-i18n-css/blob/master/README.md) into Japanese.

## Ja

* [ ] [index.html](https://github.com/mirai-audio/system-font-i18n-css/blob/master/index.html)の「About」セクションを日本語に翻訳してください。
* [ ] [index.html](https://github.com/mirai-audio/system-font-i18n-css/blob/master/index.html)の 「Usage」セクションを日本語に翻訳してください。
* [ ] [index.html](https://github.com/mirai-audio/system-font-i18n-css/blob/master/index.html)の 「Support」セクションを日本語に翻訳してください。
* [ ] [README](https://github.com/mirai-audio/system-font-i18n-css/blob/master/README.md)を日本語に翻訳してください。

**Comment by Kijimu7** [comment]
Hi, I am a native Japanese speaker. Do you want me to translate in Japanese?

**Comment by 0xadada** [comment]
おねがい!

-------- Original Message --------

**Comment by 0xadada** [comment]
@Kijimu7 おねがい!

**Comment by Kijimu7** [comment]
Translate About, Usage and Support to Japanese?

**Comment by 0xadada** [comment]
@Kijimu7 yes, in `index.html`, perhaps make `index.jp.html` for the translation file. The `README` also needs translation.

**Comment by Kijimu7** [comment]
Ok, should I go to clone/download and translate?

**Comment by Kijimu7** [comment]
I meant download index.html and make index.jp.html

**Comment by 0xadada** [comment]
@Kijimu7 yes please, clone and translate. Then open a PR and i'll work with you to help merge it 🙇

**Comment by Kijimu7** [comment]
I don't think I can translate them all, but I will translate as much I can.

**Comment by 0xadada** [comment]
This is still available and up for grabs!
